cypress/e2e/release-gate/landing-page.cy.ts

```diff
@@ -1,8 +1,258 @@
+// Declared variables to increase reusability if making breaking change!
+const lastVisitedJSON =
```

Comment:
Is this an e2e test or a component test? If it's a component-level test then hard-coded JSON is probably fine. For an e2e test, I would expect the data to be produced through user interactions with the page.

Reply:
I had this question before. If nothing changed from then, read Martin's comment above and ping him if it should be done differently --- "We will need a custom account for this. We are currently using the common qa account. Meaning that any changes we make, will have a persistent impact which might mess with other e2e testing that is happening. I'd say, let's switch but only once we have a unique account for these tests."

Reply:
@catastrophe-brandon we need a new testing account. We have run into issues that were false negative results because during the test run someone else was using the testing account in the same area. Once we have one, we can remove the intercept definitions.